action.yml

```diff
@@ -8,6 +8,9 @@ inputs:
   path:
     description: 'Destination path'
     required: false
+  artifact-folder:
```

Comment:
We should go with a verb here since it's a boolean value. Maybe `create-download-folder`?

I don't think we need to repeat "artifact" since this is `download-artifact`